staging/src/k8s.io/kubectl/pkg/cmd/completion/completion.go

```diff
@@ -167,7 +167,7 @@ func runCompletionBash(out io.Writer, boilerPlate string, kubectl *cobra.Command
 		return err
 	}
 
-	return kubectl.GenBashCompletion(out)
+	return kubectl.GenBashCompletionV2(out, false)
```

Comment:
Maybe put a comment that Cobra 1.3 is needed before enabling descriptions?

Reply:
Will do.

By the way

Reply:
Thanks.  I added a comment.

By the way, I looked into doing the upgrade, but it seems like the build wants to upgrade cobra across the entire k/k repo, not just kubectl.  I decided to defer doing the upgrade until another time.

Reply:
Yeah, it seems like updating Cobra is more work than it first seems when looking at #103448 for example.
@eddiezane has been suffering through those upgrades.

On a self-serving basis, moving to Cobra 1.3 would also help fix my proposed solution for plugin completions of #105867 😄